Boundaries.geojson

```diff
@@ -51,25 +51,36 @@
556, 54.029166666667 ], [ 0.45527777777778, 54.028055555556 ], [ 2.6683333333333, 53.383888888889 ], [ 2.7333333333333, 53.303611111111 ], [ 3, 53.268888888889 ], [ 3.1838888888889, 53.244722222222 ], [ 3.3661111111111, 53.174722222222 ], [ 2.89925, 52.617736111111 ], [ 2.7788888888889, 52.471944444444 ], [ 2.7102777777778, 52.388611111111 ], [ 2.6438888888889, 52.307222222222 ], [ 2.4880555555556, 52.114722222222 ], [ 2.3562833333333, 51.9506 ], [ 2.3579611111111, 51.952697222222 ], [ 2.6669083333333, 51.974172222222 ], [ 3.1718361111111, 52.007513888889 ], [ 3.1718361111111, 51.965677777778 ], [ 3.1718361111111, 51.480602777778 ], [ 2.6669083333333, 51.598477777778 ], [ 2.5, 51.636830555556 ], [ 2.5, 51.455588888889 ], [ 2, 51.5 ], [ 2, 51.383333333333 ], [ 2, 51.116666666667 ], [ 1.825, 51.079166666667 ], [ 1.7625, 51.065 ], [ 1.4666666666667, 51 ], [ 1.465, 50.905277777778 ], [ 1.7347222222222, 50.712222222222 ], [ 1.6363888888889, 50.656944444444 ], [ 1.5383333333333, 50.601666666667 ], [ 1.9516666666667, 50.229444444444 ], [ 1.8586111111111, 50.171388888889 ], [ 1.7575, 50.116388888889 ], [ 1.6516666666667, 50.058611111111 ], [ 1.4525, 49.948888888889 ], [ 1.4033333333333, 49.921944444444 ], [ 1.3288888888889, 49.978055555556 ], [ 0.76972222222222, 50.401388888889 ], [ 0.76972222222222, 50.401388888889 ], [ -0.25, 50 ], [ -0.53333333333333, 50 ], [ -1.1580555555556, 50 ], [ -1.7169444444444, 50 ], [ -1.7833333333333, 50 ], [ -2, 50 ], [ -2, 49.5 ], [ -1.6666666666667, 49.033333333333 ], [ -3, 49.033333333333 ], [ -3, 49.830833333333 ], [ -3.1712066666667, 49.797341388889 ], [ -4.1605555555556, 49.621944444444 ], [ -4.45, 49.566666666667 ], [ -4.9163888888889, 49.478055555556 ], [ -6.9380555555556, 49.556388888889 ], [ -8, 49.583333333333 ], [ -8, 51 ], [ -7.2308333333333, 51.424444444444 ], [ -6.1222222222222, 52.013611111111 ], [ -5.8422222222222, 52.158611111111 ], [ -5.5007022222222, 52.333518055556 ], [ -5.5, 52.773333333333 ], [ -5.5, 52.951388888889 ], [ -5.5, 53.196666666667 ], [ -5.5, 53.398055555556 ], [ -5.5, 53.804722222222 ], [ -5.5, 53.813055555556 ], [ -5.1641666666667, 53.803611111111 ], [ -5.1113888888889, 54.0275 ], [ -5.0525, 54.051111111111 ], [ -5.0516472222222, 54.061440833333 ], [ -5.0482472222222, 54.073651388889 ], [ -5.0444444444444, 54.081944444444 ], [ -5.1688888888889, 54.133055555556 ], [ -5.0058333333333, 54.269166666667 ], [ -4.8127777777778, 54.189444444444 ], [ -4.4452777777778, 54.238055555556 ], [ -4.4225466666667, 54.240355833333 ], [ -4.4015013888889, 54.241326944444 ], [ -4.3803911111111, 54.241218888889 ], [ -4.3593780555556, 54.2400325 ], [ -4.3387430555556, 54.237776666667 ], [ -4.3182886111111, 54.234692222222 ], [ -4.2985277777778, 54.230134722222 ], [ -4.2794927777778, 54.224807222222 ], [ -4.2611111111111, 54.218333333333 ], [ -3.5930555555556, 54.502777777778 ], [ -3.5513888888889, 54.25 ], [ -3.0830619444444, 54.25 ], [ -2.1666666666667, 54.25 ], [ -1.9338786111111, 54.943971111111 ] ] ] ] } },
```

Comment:
So, I will amend just the EGTT definition to exclude these areas, leaving the sector boundaries as is?

Reply:
Problem is, when controlling LON_CTR, these areas will not light up. This then defeats the whole purpose of defining them.

Reply:
The only thing I think that could be done with the Channel Islands, is to set LFRR as the "proper" FIR boundary, it will then draw a line across the top, whilst preserving the activation when LON_CTR is on.

Reply:
Fixed in 2103d7994f79713f7f6e5557ffc28ca9509b2869

![image](https://github.com/vatsimnetwork/vatspy-data-project/assets/13762210/38ff82f1-fee6-4bf9-9b4d-d4ebe3f998d7)

Reply:
Fixed in 2103d7994f79713f7f6e5557ffc28ca9509b2869

![image](https://github.com/vatsimnetwork/vatspy-data-project/assets/13762210/38ff82f1-fee6-4bf9-9b4d-d4ebe3f998d7)